libs/ec/inc/public/tfc/devices/util.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <units/isq/si/electric_current.h>
```

Comment:
you should not need any include here

maybe add the mapping function to a namespace

Reply:
Will be used later.